### PR TITLE
Add psalm annotations

### PR DESCRIPTION
**Pull request by Fantom409**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: src/ArrayHelper.php [hunk 1]
```diff
@@ -261,7 +284,7 @@ public static function getValue($array, $key, $default = null)
     }
 
     /**
-     * @param array|object $array array or object to extract value from
+     * @param mixed $array array or object to extract value from, otherwise method will return $default
```
Comment (vjik) [inline src/ArrayHelper.php:287]: Why mixed?

Comment (Fantom409) [reply src/ArrayHelper.php:287]: Because it will return $default if $array is not array and not object

Comment (vjik) [reply src/ArrayHelper.php:287]: I think, if `$array` is not array and not object, should throws `InvalidArgumentException`.

**Comment by vjik** [comment]
Static analysis is fail... maybe increade pslam error level ?

**Comment by Fantom409** [comment]
> Static analysis is fail... maybe increade pslam error level ?

To fix this, more annotations needed, I'm working on this

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**